pkg/sql/logictest/testdata/logic_test/udf

```diff
@@ -1211,6 +1211,66 @@ CREATE FUNCTION public.f_test_cor(IN a INT8, IN b INT8)
   SELECT 3;
 $$
 
+subtest udt_rewrite
+
+statement ok
+CREATE FUNCTION f_udt_rewrite() RETURNS notmyworkday LANGUAGE SQL AS $$ SELECT 'Monday':: notmyworkday $$;
+
+query T
+SELECT @2 FROM [SHOW CREATE FUNCTION f_udt_rewrite];
+----
+CREATE FUNCTION public.f_udt_rewrite()
+    RETURNS test.public.notmyworkday
+    VOLATILE
+    NOT LEAKPROOF
+    CALLED ON NULL INPUT
+    LANGUAGE SQL
+    AS $$
+    SELECT b'@':::@100115;
```

Comment:
oops.. this is bad.. need to reverse the rewrite when doing `show create`...
will fix this

Reply:
this is fixed partially in the last commit. also created #87475 to track the issue.